### ISSUE TITLE
Unable to see newly added files in GitHub Desktop in the cloned local folder from GitHub

### ISSUE THREAD

**Issue by iftikhar-husain** [desktop/desktop#10049]
### Describe the bug

Unable to see newly added files in GitHub Desktop in the cloned local folder from GitHub.

### Version & OS

GitHub Ver 2.5.2
OS Ver Windows 10
### Steps to reproduce the behavior

1. Cloned an existing repository from github to local system using GitHub desktop
2. Added new stuff (files/folders) under the cloned folder structure
3. No changes showing within the GitHub Desktop so that I can add newly added stuff to web repository and push/commit it

### Actual behavior

A clear and concise description of what actually happened.

### Screenshots

Add screenshots to help explain your problem, if applicable.

### Logs

Attach your logs by opening the `Help` menu and selecting `Show Logs...`, if applicable.

**Comment by steveward** [comment]
@iftikhar-husain thanks for the report. A few questions:
- Could you share a screenshot of what you are seeing in GitHub Desktop? 
- Can you verify that your files exist in the location when you go to `Repository` > `Open in Explorer`? 
- Are you ignoring any files in this repository? You can check this by going to `Repository` > `Repository Settings` > `Ignored Files`.

**Comment by iftikhar-husain** [comment]
Thank you Steve for contacting, actually I resolved the problem by some guidelines from the Internet.

The problem was due the .git folders. There were two folders present in the same directory hierarchy. One created by GitHub desktop after repo clone and the other one was I created by expo platform (React Native) that I am using. I deleted the nested .git folder created by expo while generating the project stuff. This resolved the issue.

However I would suggest in these situations the GitHub desktop should give some sort of relevant error message.

Thanks,
Iftikhar

Syed Iftikhar Hussain
